### PR TITLE
Adding sai_samplepacket_type_t to sai_attribute_value_t 

### PR DESCRIPTION
**Pull request by atitjain**
Adding sai_samplepacket_type_t to sai_attribute_value_t  so that attribute could be set for samplepacket API.

### CODE REVIEW

File: sai/inc/saitypes.h [hunk 1]
```diff
@@ -337,6 +337,7 @@ typedef union {
     sai_acl_field_data_t aclfield;
     sai_acl_action_data_t aclaction;
     sai_port_breakout_t portbreakout;
+    sai_samplepacket_type_t sflowtype;
```
Comment (lguohan) [inline sai/inc/saitypes.h]: sai_samplepacket_type_t is a enum, then we should just use sai_int32_t s32 to pass the value.

Comment (atitjain) [inline sai/inc/saitypes.h]: I see, let me check. Thanks.

**Comment by lguohan** [comment]
can we close this issue now?

**Comment by atitjain** [comment]
Closing.

**Pull request closed without merging**